modules/update-upgrading-cli.adoc

```diff
@@ -46,12 +46,12 @@ $ oc adm upgrade
 ----
 +
 .Example output
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
 Cluster version is 4.9.23
 
 Upstream is unset, so the cluster will use an appropriate default.
-Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.9, fast-4.10, fast-4.9, stable-4.10, stable-4.9)
+Channel: stable-{product-version} (available channels: candidate-{product-version}, fast-{product-version}, stable-{product-version})
```

Comment:
We're currently including `eus-...` in our channel recommendations below, so we probably want to include `eus-...` somehow in this line.  But we probably don't want to use `eus-{product-version}`, because only 4.even are EUS (e.g. both 4.11 and 4.12 docs builds would want to use `eus-4.12`).  Not sure if it's worth defining `{eus-inclusion-version}` or something?